artifact/image/layerscanning/image/file_node.go

```diff
@@ -94,6 +94,19 @@ func (f *fileNode) ReadAt(b []byte, off int64) (n int, err error) {
 	return f.file.ReadAt(b, off)
 }
 
+func (f *fileNode) Seek(offset int64, whence int) (n int64, err error) {
+	if f.isWhiteout {
+		return 0, fs.ErrNotExist
+	}
+	if f.file == nil {
+		f.file, err = os.Open(f.RealFilePath())
```

Comment:
do we need to close?

Reply:
Nope, it'll get closed when someone calls Close() on the fileNode